crates/node-builder/src/builder.rs

```diff
@@ -665,6 +666,17 @@ where
         let network_client = network.fetch_client().await?;
         let (consensus_engine_tx, mut consensus_engine_rx) = unbounded_channel();
 
+        if let Some(skip_fcu_threshold) = config.debug.skip_fcu {
+            debug!(target: "reth::cli", "spawning skip FCU task");
+            let (skip_fcu_tx, skip_fcu_rx) = unbounded_channel();
+            let engine_skip_fcu = EngineApiSkipFcu::new(skip_fcu_threshold);
+            executor.spawn_critical(
+                "skip FCU interceptor",
+                engine_skip_fcu.intercept(consensus_engine_rx, skip_fcu_tx),
+            );
+            consensus_engine_rx = skip_fcu_rx;
+        }
+
```

Comment:
should this be after the engine api store interceptor? in case you wanted to use both

Reply:
the primary purpose of engine API store is replay. my thinking here is that you'd want to replay exactly the messages that reached the engine, alternatively we could add support for this arg in the replayer as well